### PR TITLE
Cleanup 1

### PR DESCRIPTION
**Pull request by PLASMAchicken**

### CODE REVIEW

File: src/BPEssentials/Utils/ChatUtils.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+using BPEssentials.Enums;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.API.ExtensionMethods;
+using BrokeProtocol.Collections;
+using BrokeProtocol.Entities;
+
+
+namespace BPEssentials.Utils
+{
+    public static class ChatUtils
```
Comment (UserR00T) [inline src/BPEssentials/Utils/ChatUtils.cs]: I suggest changing this to `Chat`.
```suggestion
    public static class Chat
```

File: src/BPEssentials/Utils/LevenshteinDistanceUtil.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace BPEssentials.Utils
+{
+    public static class LevenshteinDistanceUtil
```
Comment (UserR00T) [inline src/BPEssentials/Utils/LevenshteinDistanceUtil.cs]: I suggest changing this to `LevenshteinDistance`.
```suggestion
    public static class LevenshteinDistance
```

File: src/BPEssentials/Utils/ReloadUtil.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿namespace BPEssentials.Utils
+{
+    public static class ReloadUtil
```
Comment (UserR00T) [inline src/BPEssentials/Utils/ReloadUtil.cs]: I suggest changing this to `IO`, `File`, or `FileUtil`.
```suggestion
    public static class IO
```

**Comment by UserR00T** [comment]
Also if possible, try to resolve the Codacy issues. Most are simple ones like curly brackets and stuff.

File: src/BPEssentials/Utils/ChatUtils.cs (hunk 1 above)
Comment (PLASMAchicken) [reply src/BPEssentials/Utils/ChatUtils.cs]: The Enums are also called Chat, so I would prefer to keep this name

**Pull request merged**